app/lib/main.dart

```diff
@@ -56,11 +66,14 @@ class AccountHeader extends StatefulWidget {
 
   @override
   _AccountHeaderState createState() => _AccountHeaderState(
-      _client, _client.displayName(), _client.userId(), _client.avatar().then((fb) => fb.toUint8List()));
+      _client,
+      _client.displayName(),
+      _client.userId(),
+      _client.avatar().then((fb) => fb.toUint8List()));
```

Comment:
okay, this a second time I find an unrelated refactoring without changes in a PR,  I'll file that as a bug now: https://github.com/effektio/effektio/issues/29 .